pyserum/market/market.py

```diff
@@ -156,12 +156,14 @@ def settle_funds(  # pylint: disable=too-many-arguments
         min_bal_for_rent_exemption = (
             self._conn.get_minimum_balance_for_rent_exemption(165)["result"] if should_wrap_sol else 0
         )  # value only matters if should_wrap_sol
+        signers = [owner]
```

Comment:
I wonder if this code change is the same as before?

Reply:
You mean as in `market/core.py`? That `signers` is not used anywhere, it is needed at this level for `self._conn.send_transaction`.